docs/hmrc-ch-mapping-fix.md

```
@@ -0,0 +1,1389 @@
+|---|---|
+| Company name | `Prime Marque Services Limited Trading As: Rainbow Care Solutions Hertfordshire` (HMRC label) |
+| Status | **Dissolved** |
+| Location | Stevenage, Hertfordshire |
+| Incorporated | 25 September 2015 |
+| Last accounts filed | 30 January 2020 |
+| Registration No. | `09794326` |
+| Registered address | Alpha House, Tipton Street, Sedgley, West Midlands, DY3 1HE |
+
+### What's actually in our DB
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)